resequencer/resquencer.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/http;
+
+type Approval record {
+    int leadLevel;
+    int score;
+    int nextPromotionLevel;
+};
+
+map<Approval[]> approvalsMap = {};
+
+final http:Client hrClient = check new ("http://api.wso2hr.com.balmock.io");
+
+service /api/v1 on new http:Listener(8080) {
+
+    resource function post employees/[string emoloyeeId]/approval(@http:Payload Approval approval) returns error? {
+        if approvalsMap.hasKey(emoloyeeId) {
```

Comment:
refactor so the if dosn't cover everything.

Reply:
Updated.